<!DOCTYPE html>
<html>

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <title>Beijing Wu | PUBLICATIONS</title>
    <meta name="description" content="Beijing Wu | Publications">

    <!-- Bootstrap & MDB -->
    <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" rel="stylesheet" integrity="sha512-MoRNloxbStBcD8z3M/2BmnT+rg4IsMxPkXaGh2zD6LGNNFE80W3onsAhRcMAMrSoyWL9xD7Ert0men7vR8LUZg==" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/css/mdb.min.css" integrity="sha512-RO38pBRxYH3SoOprtPTD86JFOclM51/XTIdEPh5j8sj4tp8jmQIx26twG52UaLi//hQldfrh7e51WzP9wuP32Q==" crossorigin="anonymous" />

    <!-- Fonts & Icons -->
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css" integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog==" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.9.0/css/academicons.min.css" integrity="sha512-W4yqoT1+8NLkinBLBZko+dFB2ZbHsYLDdr50VElllRcNt2Q4/GSs6u71UHKxB7S6JEMCp5Ve4xjh3eGQl/HRvg==" crossorigin="anonymous">
    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">

    <!-- Code Syntax Highlighting -->
    <link rel="stylesheet" href="https://gitcdn.link/repo/jwarby/jekyll-pygments-themes/master/github.css" />

    <!-- Styles -->
    <link rel="shortcut icon" href="/assets/img/favicon.ico">
    <link rel="stylesheet" href="/assets/css/main.css">

    <link rel="canonical" href="/">

    <!-- Theming-->
    <script src="/assets/js/theme.js"></script>

    <!-- Start of Google Analytics code ID -->
    <!-- Global site tag (gtag.js) - Google Analytics -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-162315295-1"></script>
    <script>
        window.dataLayer = window.dataLayer || [];

        function gtag() {
            dataLayer.push(arguments);
        }
        gtag('js', new Date());

        gtag('config', 'UA-162315295-1');
    </script>
    <!-- End of Google Analytics code ID -->
</head>

<body class="fixed-top-nav">

    <!-- Header -->
    <header>

        <!-- Nav Bar -->
        <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
            <div class="container">

                <!-- Social Icons -->
                <div class="row ml-1 ml-sm-0">
                    <span class="contact-icon text-center">

              <a href="https://github.com/wubeijing" target="_blank" title="GitHub"><i class="fab fa-github"></i></a>
              <a href="https://www.linkedin.com/in/beijing-wu-39aabb91/" target="_blank" title="LinkedIn"><i class="fab fa-linkedin"></i></a>
              <a href="/assets/pdf/BeijingWu_2020_v02.pdf" target="_blank" title="Resume"><i class="fab fa-readme"></i></a>
            
            </span>
                </div>

                <!-- Navbar Toogle -->
                <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar top-bar"></span>
            <span class="icon-bar middle-bar"></span>
            <span class="icon-bar bottom-bar"></span>
          </button>

                <div class="collapse navbar-collapse text-right" id="navbarNav">
                    <ul class="navbar-nav ml-auto flex-nowrap">

                        <!-- About -->
                        <li class="nav-item">
                            <a class="nav-link" href="/">ABOUT</a>
                        </li>

                        <!-- Projects -->
                        <li class="nav-item ">
                            <a class="nav-link" href="/projects/">PROJECTS</a>
                        </li>

                        <!-- Skills -->
                        <li class="nav-item ">
                            <a class="nav-link" href="/skills/">SKILLS</a>
                        </li>

                        <!-- Academics -->
                        <li class="nav-item ">
                            <a class="nav-link" href="/education/">ACADEMICS</a>
                        </li>

                        <!-- Publications -->
                        <li class="nav-item active">
                            <a class="nav-link" href="/publication/">PUBLICATIONS<span class="sr-only">(current)</span></a>
                        </li>

                        <div class="toggle-container">
                            <a id="light-toggle">
                                <i class="fas fa-moon"></i>
                                <i class="fas fa-sun"></i>
                            </a>
                        </div>

                    </ul>
                </div>
            </div>
        </nav>
    </header>

    <!-- Content -->
    <div class="container mt-5">
        <div class="publications">

            <header class="post-header">
                <h1 class="post-title">Publications</h1>
                <p class="post-description">* indicates equal contribution.</p>
            </header>

            
            <div class="projects">
    
                <h2 class="year">2019</h2>
                <ol class="bibliography">
                <!-- publication 1 -->
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr">
                                <abbr class="badge" style="text-align: center; display: block">Nature Genetics</abbr>
                            </div>

                            <div id="tenney2020lit" class="col-sm-8">
                                <a href="https://www.nature.com/articles/s41588-019-0505-9" target="_blank">
                                    <div class="title"><b>Landscape of stimulation-responsive chromatin across diverse human immune cells</b></div>
                                </a>

                                <div class="author">
                                    Diego Calderon, Michelle L. T. Nguyen, Anja Mezger, Arwa Kathiria, Fabian Müller, Vinh Nguyen, Ninnia Lescano, <b><u>Beijing Wu</u></b>, John Trombetta, Jessica V. Ribado, David A. Knowles, Ziyue Gao, Franziska Blaeschke, Audrey V. Parent, Trevor D. Burt, Mark S. Anderson, Lindsey A. Criswell, William J. Greenleaf, Alexander Marson & Jonathan K. Pritchard
                                </div>

                                <div class="periodical">
                                    <b><i>Nature Genetics</i></b>    51, pages1494–1505 (2019)
                                </div>

                                <div class="links">
                                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                                    <a href="https://wubeijing.github.io/assets/pdf/nihms-1538431.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">Paper</a>
                                </div>

                                <!-- Hidden abstract block -->
                                <div class="abstract hidden">
                                    <p>A hallmark of the immune system is the interplay among specialized cell types transitioning between resting and stimulated states. The gene regulatory landscape of this dynamic system has not been fully characterized in human cells. Here we collected assay for transposase-accessible chromatin using sequencing (ATAC-seq) and RNA sequencing data under resting and stimulated conditions for up to 32 immune cell populations. Stimulation caused widespread chromatin remodeling, including response elements shared between stimulated B and T cells. Furthermore, several autoimmune traits showed significant heritability in stimulation-responsive elements from distinct cell types, highlighting the importance of these cell states in autoimmunity. Allele-specific read mapping identified variants that alter chromatin accessibility in particular conditions, allowing us to observe evidence of function for a candidate causal variant that is undetected by existing large-scale studies in resting cells. Our results provide a resource of chromatin dynamics and highlight the need to characterize the effects of genetic variation in stimulated cells.</p>
                                </div>
                            </div>
                        </div>
                    </li>
                    
                <h2 class="year">2018</h2>    
                <!-- publication 2 -->
                <li>
                    <div class="row">
                        <div class="col-sm-2 abbr">
                            <abbr class="badge" style="text-align: center; display: block">Cell</abbr>
                        </div>

                        <div id="tenney2020lit" class="col-sm-8">
                            <a href="https://www.cell.com/cell/fulltext/S0092-8674(18)31514-9?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867418315149%3Fshowall%3Dtrue" target="_blank">
                                <div class="title"><b>Coupled Single-Cell CRISPR Screening and Epigenomic Profiling Reveals Causal Gene Regulatory Networks</b></div>
                            </a>

                            <div class="author">
                                Adam J. Rubin<sup>*</sup>, Kevin R. Parker<sup>*</sup>, Ansuman T. Satpathy<sup>*</sup>, Yanyan Qi, <b><u>Beijing Wu</u></b>, Alvin J. Ong, Maxwell R. Mumbach, Andrew L. Ji, Daniel S. Kim, Seung Woo Cho, Brian J. Zarnegar, William J. Greenleaf, Howard Y. Chang & Paul A. Khavari
                            </div>

                            <div class="periodical">
                                <b><i>Cell</i></b>    VOLUME 176, ISSUE 1-2, P361-376.E17 (2018)
                            </div>

                            <div class="links">
                                <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                                <a href="https://wubeijing.github.io/assets/pdf/nihms-1513141.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">Paper</a>
                            </div>

                            <!-- Hidden abstract block -->
                            <div class="abstract hidden">
                                <p>Here, we present Perturb-ATAC, a method that combines multiplexed CRISPR interference or knockout with genome-wide chromatin accessibility profiling in single cells based on the simultaneous detection of CRISPR guide RNAs and open chromatin sites by assay of transposase-accessible chromatin with sequencing (ATAC-seq). We applied Perturb-ATAC to transcription factors (TFs), chromatin-modifying factors, and noncoding RNAs (ncRNAs) in ∼4,300 single cells, encompassing more than 63 genotype-phenotype relationships. Perturb-ATAC in human B lymphocytes uncovered regulators of chromatin accessibility, TF occupancy, and nucleosome positioning and identified a hierarchy of TFs that govern B cell state, variation, and disease-associated cis-regulatory elements. Perturb-ATAC in primary human epidermal cells revealed three sequential modules of cis-elements that specify keratinocyte fate. Combinatorial deletion of all pairs of these TFs uncovered their epistatic relationships and highlighted genomic co-localization as a basis for synergistic interactions. Thus, Perturb-ATAC is a powerful strategy to dissect gene regulatory networks in development and disease.</p>
                            </div>
                        </div>
                    </div>
                </li>

                <!-- publication 3 -->
                <li>
                    <div class="row">
                        <div class="col-sm-2 abbr">
                            <abbr class="badge" style="text-align: center; display: block">Cell</abbr>
                        </div>

                        <div id="tenney2020lit" class="col-sm-8">
                            <a href="https://www.cell.com/cell/fulltext/S0092-8674(18)30446-X?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS009286741830446X%3Fshowall%3Dtrue" target="_blank">
                                <div class="title"><b>Integrated Single-Cell Analysis Maps the Continuous Regulatory Landscape of Human Hematopoietic Differentiation</b></div>
                            </a>

                            <div class="author">
                                Jason D. Buenrostro, M. Ryan Corces, Caleb A. Lareau, <b><u>Beijing Wu</u></b>, Alicia N. Schep, Martin J. Aryee, Ravindra Majeti, Howard Y. Chang & William J. Greenleaf
                            </div>

                            <div class="periodical">
                                <b><i>Cell</i></b>    VOLUME 173, ISSUE 6, P1535-1548.E16 (2018)
                            </div>

                            <div class="links">
                                <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                                <a href="https://wubeijing.github.io/assets/pdf/nihms963511.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">Paper</a>
                            </div>

                            <!-- Hidden abstract block -->
                            <div class="abstract hidden">
                                <p>Human hematopoiesis involves cellular differentiation of multipotent cells into progressively more lineage-restricted states. While the chromatin accessibility landscape of this process has been explored in defined populations, single-cell regulatory variation has been hidden by ensemble averaging. We collected single-cell chromatin accessibility profiles across 10 populations of immunophenotypically defined human hematopoietic cell types and constructed a chromatin accessibility landscape of human hematopoiesis to characterize differentiation trajectories. We find variation consistent with lineage bias toward different developmental branches in multipotent cell types. We observe heterogeneity within common myeloid progenitors (CMPs) and granulocyte-macrophage progenitors (GMPs) and develop a strategy to partition GMPs along their differentiation trajectory. Furthermore, we integrated single-cell RNA sequencing (scRNA-seq) data to associate transcription factors to chromatin accessibility changes and regulatory elements to target genes through correlations of expression and regulatory element accessibility. Overall, this work provides a framework for integrative exploration of complex regulatory dynamics in a primary human tissue at single-cell resolution.</p>
                            </div>
                        </div>
                    </div>
                </li>

                <!-- publication 4 -->
                <li>
                    <div class="row">
                        <div class="col-sm-2 abbr">
                            <abbr class="badge" style="text-align: center; display: block">Cell</abbr>
                        </div>

                        <div id="tenney2020lit" class="col-sm-8">
                            <a href="https://www.nature.com/articles/s41591-018-0008-8" target="_blank">
                                <div class="title"><b>Transcript-indexed ATAC-seq for precision immune profiling</b></div>
                            </a>

                            <div class="author">
                                Ansuman T. Satpathy<sup>*</sup>, Naresha Saligrama<sup>*</sup>, Jason D. Buenrostro<sup>*</sup>, Yuning Wei, <b><u>Beijing Wu</u></b>, Adam J. Rubin, Jeffrey M. Granja, Caleb A. Lareau, Rui Li, Yanyan Qi, Kevin R. Parker, Maxwell R. Mumbach, William S. Serratelli, David G. Gennert, Alicia N. Schep, M. Ryan Corces, Michael S. Khodadoust, Youn H. Kim, Paul A. Khavari, William J. Greenleaf, Mark M. Davis & Howard Y. Chang
                            </div>

                            <div class="periodical">
                                <b><i>Nature Medicine</i></b>    24, pages580–590 (2018)
                            </div>

                            <div class="links">
                                <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                                <a href="https://wubeijing.github.io/assets/pdf/nihms941149.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">Paper</a>
                            </div>

                            <!-- Hidden abstract block -->
                            <div class="abstract hidden">
                                <p>T cells create vast amounts of diversity in their T cell receptor (TCR) genes, enabling individual clones to recognize specific peptide-MHC ligands. Here we combine TCR sequencing and assay for transposase-accessible chromatin analysis at the single-cell level to provide information on the TCR specificity and epigenomic state of individual T cells. Using this approach, termed Transcript-indexed ATAC-seq (T-ATAC-seq), we identify epigenomic signatures in immortalized leukemic T cells, primary human T cells from healthy volunteers, and primary leukemic T cells from patient samples. In healthy peripheral blood CD4<sup>+</sup> T cells, we identify <i>cis</i> and <i>trans</i> regulators of naive and memory T cell states and find substantial heterogeneity in surface markerdefined T cell populations. In patients with cutaneous T cell lymphoma, T-ATAC-seq enabled identification of leukemic and non-leukemic regulatory pathways in T cells from the same individual, separating signals arising from the malignant clone from background T cell noise. Thus, T-ATAC-seq is a new tool that enables analysis of epigenomic landscapes in clonal T cells and should be valuable for studies of T cell malignancy, immunity, and immunotherapy.</p>
                            </div>
                        </div>
                    </div>
                </li>

                

                </ol>
            </div>
        </div>
            
    </div>

    <!-- Footer -->
    <footer class="fixed-bottom">
        <div class="container mt-0">
            &copy; Powered by <a href="http://jekyllrb.com/" target="_blank">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio">al-folio</a> theme. Additional schema modifications contributed by <a href="https://iftenney.github.io/">others</a>.
            Hosted by <a href="https://pages.github.com/" target="_blank">GitHub Pages</a>.
        </div>
    </footer>
</body>

<!-- jQuery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg==" crossorigin="anonymous"></script>

<!-- Bootsrap & MDB scripts -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/2.4.4/umd/popper.min.js" integrity="sha512-eUQ9hGdLjBjY3F41CScH3UX+4JDSI9zXeroz7hJ+RteoCaY+GP/LDoM8AO+Pt+DRFw3nXqsjh9Zsts8hnYv8/A==" crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha512-M5KW3ztuIICmVIhjSqXe01oV2bpe248gOxqmlcYrEzAvws7Pw3z6BK0iGbrwvdrUQUhi3eXgtxp5I8PDo9YfjQ==" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/js/mdb.min.js" integrity="sha512-Mug9KHKmroQFMLm93zGrjhibM2z2Obg9l6qFG2qKjXEXkMp/VDkI4uju9m4QKPjWSwQ6O2qzZEnJDEeCw0Blcw==" crossorigin="anonymous"></script>

<!-- Load Common JS -->
<script src="/assets/js/common.js"></script>

<!-- Load DarkMode JS -->
<script src="/assets/js/dark_mode.js"></script>

</html>