<!DOCTYPE html>
<html>

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <title>Beijing Wu | PUBLICATIONS</title>
    <meta name="description" content="Beijing Wu | Publications">

    <!-- Bootstrap & MDB -->
    <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" rel="stylesheet" integrity="sha512-MoRNloxbStBcD8z3M/2BmnT+rg4IsMxPkXaGh2zD6LGNNFE80W3onsAhRcMAMrSoyWL9xD7Ert0men7vR8LUZg==" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/css/mdb.min.css" integrity="sha512-RO38pBRxYH3SoOprtPTD86JFOclM51/XTIdEPh5j8sj4tp8jmQIx26twG52UaLi//hQldfrh7e51WzP9wuP32Q==" crossorigin="anonymous" />

    <!-- Fonts & Icons -->
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css" integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog==" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.9.0/css/academicons.min.css" integrity="sha512-W4yqoT1+8NLkinBLBZko+dFB2ZbHsYLDdr50VElllRcNt2Q4/GSs6u71UHKxB7S6JEMCp5Ve4xjh3eGQl/HRvg==" crossorigin="anonymous">
    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">

    <!-- Code Syntax Highlighting -->
    <link rel="stylesheet" href="https://gitcdn.link/repo/jwarby/jekyll-pygments-themes/master/github.css" />

    <!-- Styles -->
    <link rel="shortcut icon" href="/assets/img/favicon.ico">
    <link rel="stylesheet" href="/assets/css/main.css">

    <link rel="canonical" href="/">

    <!-- Theming-->
    <script src="/assets/js/theme.js"></script>

    <!-- Start of Google Analytics code ID -->
    <!-- Global site tag (gtag.js) - Google Analytics -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-162315295-1"></script>
    <script>
        window.dataLayer = window.dataLayer || [];

        function gtag() {
            dataLayer.push(arguments);
        }
        gtag('js', new Date());

        gtag('config', 'UA-162315295-1');
    </script>
    <!-- End of Google Analytics code ID -->
</head>

<body class="fixed-top-nav">

    <!-- Header -->
    <header>

        <!-- Nav Bar -->
        <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
            <div class="container">

                <!-- Social Icons -->
                <div class="row ml-1 ml-sm-0">
                    <span class="contact-icon text-center">

              <a href="https://github.com/wubeijing" target="_blank" title="GitHub"><i class="fab fa-github"></i></a>
              <a href="https://www.linkedin.com/in/beijing-wu-39aabb91/" target="_blank" title="LinkedIn"><i class="fab fa-linkedin"></i></a>
              <a href="/assets/pdf/BeijingWu_2020_v02.pdf" target="_blank" title="Resume"><i class="fab fa-readme"></i></a>
            
            </span>
                </div>

                <!-- Navbar Toogle -->
                <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar top-bar"></span>
            <span class="icon-bar middle-bar"></span>
            <span class="icon-bar bottom-bar"></span>
          </button>

                <div class="collapse navbar-collapse text-right" id="navbarNav">
                    <ul class="navbar-nav ml-auto flex-nowrap">

                        <!-- About -->
                        <li class="nav-item">
                            <a class="nav-link" href="/">ABOUT</a>
                        </li>

                        <!-- Projects -->
                        <li class="nav-item ">
                            <a class="nav-link" href="/projects/">PROJECTS</a>
                        </li>

                        <!-- Skills -->
                        <li class="nav-item ">
                            <a class="nav-link" href="/skills/">SKILLS</a>
                        </li>

                        <!-- Academics -->
                        <li class="nav-item ">
                            <a class="nav-link" href="/education/">ACADEMICS</a>
                        </li>

                        <!-- Publications -->
                        <li class="nav-item active">
                            <a class="nav-link" href="/publication/">PUBLICATIONS<span class="sr-only">(current)</span></a>
                        </li>

                        <div class="toggle-container">
                            <a id="light-toggle">
                                <i class="fas fa-moon"></i>
                                <i class="fas fa-sun"></i>
                            </a>
                        </div>

                    </ul>
                </div>
            </div>
        </nav>
    </header>

    <!-- Content -->
    <div class="container mt-5">
        <div class="publications">

            <header class="post-header">
                <h1 class="post-title">Publications</h1>
                <p class="post-description">* indicates equal contribution.</p>
            </header>

            
            <div class="projects">
    
                <h2 class="year">2019</h2>
                <ol class="bibliography">
                <!-- publication 1 -->
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr">
                                <abbr class="badge" style="text-align: center; display: block">Nature Genetics</abbr>
                            </div>

                            <div id="tenney2020lit" class="col-sm-8">
                                <a href="https://www.nature.com/articles/s41588-019-0505-9" target="_blank">
                                    <div class="title"><b>Landscape of stimulation-responsive chromatin across diverse human immune cells</b></div>
                                </a>

                                <div class="author">
                                    Diego Calderon, Michelle L. T. Nguyen, Anja Mezger, Arwa Kathiria, Fabian Müller, Vinh Nguyen, Ninnia Lescano, <a style="color: #0076df; font-weight: bold">Beijing Wu</a>, John Trombetta, Jessica V. Ribado, David A. Knowles, Ziyue Gao, Franziska Blaeschke, Audrey V. Parent, Trevor D. Burt, Mark S. Anderson, Lindsey A. Criswell, William J. Greenleaf, Alexander Marson & Jonathan K. Pritchard
                                </div>

                                <div class="periodical">
                                    Nat Genet. 2019 Oct;51(10):1494-1505. doi: 10.1038/s41588-019-0505-9.
                                </div>

                                <div class="links">
                                    <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                                    <a href="https://wubeijing.github.io/assets/pdf/nihms-1538431.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">Paper</a>
                                </div>

                                <!-- Hidden abstract block -->
                                <div class="abstract hidden">
                                    <p align="left">A hallmark of the immune system is the interplay among specialized cell types transitioning between resting and stimulated states. The gene regulatory landscape of this dynamic system has not been fully characterized in human cells. Here we collected assay for transposase-accessible chromatin using sequencing (ATAC-seq) and RNA sequencing data under resting and stimulated conditions for up to 32 immune cell populations. Stimulation caused widespread chromatin remodeling, including response elements shared between stimulated B and T cells. Furthermore, several autoimmune traits showed significant heritability in stimulation-responsive elements from distinct cell types, highlighting the importance of these cell states in autoimmunity. Allele-specific read mapping identified variants that alter chromatin accessibility in particular conditions, allowing us to observe evidence of function for a candidate causal variant that is undetected by existing large-scale studies in resting cells. Our results provide a resource of chromatin dynamics and highlight the need to characterize the effects of genetic variation in stimulated cells.</p>
                                </div>
                            </div>
                        </div>
                    </li>
                    
                <h2 class="year">2018</h2>    
                <!-- publication 2 -->
                <li>
                    <div class="row">
                        <div class="col-sm-2 abbr">
                            <abbr class="badge" style="text-align: center; display: block">Cell</abbr>
                        </div>

                        <div id="tenney2020lit" class="col-sm-8">
                            <a href="https://www.cell.com/cell/fulltext/S0092-8674(18)31514-9?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867418315149%3Fshowall%3Dtrue" target="_blank">
                                <div class="title"><b>Coupled Single-Cell CRISPR Screening and Epigenomic Profiling Reveals Causal Gene Regulatory Networks</b></div>
                            </a>

                            <div class="author">
                                Adam J. Rubin<sup>*</sup>, Kevin R. Parker<sup>*</sup>, Ansuman T. Satpathy<sup>*</sup>, Yanyan Qi, <a style="color: #0076df; font-weight: bold">Beijing Wu</a>, Alvin J. Ong, Maxwell R. Mumbach, Andrew L. Ji, Daniel S. Kim, Seung Woo Cho, Brian J. Zarnegar, William J. Greenleaf, Howard Y. Chang & Paul A. Khavari
                            </div>

                            <div class="periodical">
                                Cell. 2019 Jan 10;176(1-2):361-376.e17. doi: 10.1016/j.cell.2018.11.022.
                            </div>

                            <div class="links">
                                <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                                <a href="https://wubeijing.github.io/assets/pdf/nihms-1513141.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">Paper</a>
                            </div>

                            <!-- Hidden abstract block -->
                            <div class="abstract hidden">
                                <p align="left">Here, we present Perturb-ATAC, a method that combines multiplexed CRISPR interference or knockout with genome-wide chromatin accessibility profiling in single cells based on the simultaneous detection of CRISPR guide RNAs and open chromatin sites by assay of transposase-accessible chromatin with sequencing (ATAC-seq). We applied Perturb-ATAC to transcription factors (TFs), chromatin-modifying factors, and noncoding RNAs (ncRNAs) in ∼4,300 single cells, encompassing more than 63 genotype-phenotype relationships. Perturb-ATAC in human B lymphocytes uncovered regulators of chromatin accessibility, TF occupancy, and nucleosome positioning and identified a hierarchy of TFs that govern B cell state, variation, and disease-associated cis-regulatory elements. Perturb-ATAC in primary human epidermal cells revealed three sequential modules of cis-elements that specify keratinocyte fate. Combinatorial deletion of all pairs of these TFs uncovered their epistatic relationships and highlighted genomic co-localization as a basis for synergistic interactions. Thus, Perturb-ATAC is a powerful strategy to dissect gene regulatory networks in development and disease.</p>
                            </div>
                        </div>
                    </div>
                </li>

                <!-- publication 3 -->
                <li>
                    <div class="row">
                        <div class="col-sm-2 abbr">
                            <abbr class="badge" style="text-align: center; display: block">Cell</abbr>
                        </div>

                        <div id="tenney2020lit" class="col-sm-8">
                            <a href="https://www.cell.com/cell/fulltext/S0092-8674(18)30446-X?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS009286741830446X%3Fshowall%3Dtrue" target="_blank">
                                <div class="title"><b>Integrated Single-Cell Analysis Maps the Continuous Regulatory Landscape of Human Hematopoietic Differentiation</b></div>
                            </a>

                            <div class="author">
                                Jason D. Buenrostro, M. Ryan Corces, Caleb A. Lareau, <a style="color: #0076df; font-weight: bold">Beijing Wu</a>, Alicia N. Schep, Martin J. Aryee, Ravindra Majeti, Howard Y. Chang & William J. Greenleaf
                            </div>

                            <div class="periodical">
                                Cell. 2018 May 31;173(6):1535-1548.e16. doi: 10.1016/j.cell.2018.03.074.
                            </div>

                            <div class="links">
                                <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                                <a href="https://wubeijing.github.io/assets/pdf/nihms963511.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">Paper</a>
                            </div>

                            <!-- Hidden abstract block -->
                            <div class="abstract hidden">
                                <p align="left">Human hematopoiesis involves cellular differentiation of multipotent cells into progressively more lineage-restricted states. While the chromatin accessibility landscape of this process has been explored in defined populations, single-cell regulatory variation has been hidden by ensemble averaging. We collected single-cell chromatin accessibility profiles across 10 populations of immunophenotypically defined human hematopoietic cell types and constructed a chromatin accessibility landscape of human hematopoiesis to characterize differentiation trajectories. We find variation consistent with lineage bias toward different developmental branches in multipotent cell types. We observe heterogeneity within common myeloid progenitors (CMPs) and granulocyte-macrophage progenitors (GMPs) and develop a strategy to partition GMPs along their differentiation trajectory. Furthermore, we integrated single-cell RNA sequencing (scRNA-seq) data to associate transcription factors to chromatin accessibility changes and regulatory elements to target genes through correlations of expression and regulatory element accessibility. Overall, this work provides a framework for integrative exploration of complex regulatory dynamics in a primary human tissue at single-cell resolution.</p>
                            </div>
                        </div>
                    </div>
                </li>

                <!-- publication 4 -->
                <li>
                    <div class="row">
                        <div class="col-sm-2 abbr">
                            <abbr class="badge" style="text-align: center; display: block">Nature Medicine</abbr>
                        </div>

                        <div id="tenney2020lit" class="col-sm-8">
                            <a href="https://www.nature.com/articles/s41591-018-0008-8" target="_blank">
                                <div class="title"><b>Transcript-indexed ATAC-seq for precision immune profiling</b></div>
                            </a>

                            <div class="author">
                                Ansuman T. Satpathy<sup>*</sup>, Naresha Saligrama<sup>*</sup>, Jason D. Buenrostro<sup>*</sup>, Yuning Wei, <a style="color: #0076df; font-weight: bold">Beijing Wu</a>, Adam J. Rubin, Jeffrey M. Granja, Caleb A. Lareau, Rui Li, Yanyan Qi, Kevin R. Parker, Maxwell R. Mumbach, William S. Serratelli, David G. Gennert, Alicia N. Schep, M. Ryan Corces, Michael S. Khodadoust, Youn H. Kim, Paul A. Khavari, William J. Greenleaf, Mark M. Davis & Howard Y. Chang
                            </div>

                            <div class="periodical">
                                Nat Med. 2018 May;24(5):580-590. doi: 10.1038/s41591-018-0008-8.
                            </div>

                            <div class="links">
                                <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                                <a href="https://wubeijing.github.io/assets/pdf/nihms941149.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">Paper</a>
                            </div>

                            <!-- Hidden abstract block -->
                            <div class="abstract hidden">
                                <p align="left">T cells create vast amounts of diversity in their T cell receptor (TCR) genes, enabling individual clones to recognize specific peptide-MHC ligands. Here we combine TCR sequencing and assay for transposase-accessible chromatin analysis at the single-cell level to provide information on the TCR specificity and epigenomic state of individual T cells. Using this approach, termed Transcript-indexed ATAC-seq (T-ATAC-seq), we identify epigenomic signatures in immortalized leukemic T cells, primary human T cells from healthy volunteers, and primary leukemic T cells from patient samples. In healthy peripheral blood CD4<sup></sup> T cells, we identify <i>cis</i> and <i>trans</i> regulators of naive and memory T cell states and find substantial heterogeneity in surface markerdefined T cell populations. In patients with cutaneous T cell lymphoma, T-ATAC-seq enabled identification of leukemic and non-leukemic regulatory pathways in T cells from the same individual, separating signals arising from the malignant clone from background T cell noise. Thus, T-ATAC-seq is a new tool that enables analysis of epigenomic landscapes in clonal T cells and should be valuable for studies of T cell malignancy, immunity, and immunotherapy.</p>
                            </div>
                        </div>
                    </div>
                </li>

                <h2 class="year">2017</h2>
                <!-- publication 5 -->
                <li>
                    <div class="row">
                        <div class="col-sm-2 abbr">
                            <abbr class="badge" style="text-align: center; display: block">Nature Methods</abbr>
                        </div>

                        <div id="tenney2020lit" class="col-sm-8">
                            <a href="https://www.nature.com/articles/nmeth.4396" target="_blank">
                                <div class="title"><b>An improved ATAC-seq protocol reduces background and enables interrogation of frozen tissues</b></div>
                            </a>

                            <div class="author">
                                M. Ryan Corces, Alexandro E. Trevino, Emily G. Hamilton, Peyton G. Greenside, Nicholas A. Sinnott-Armstrong, Sam Vesuna, Ansuman T. Satpathy, Adam J. Rubin, Kathleen S. Montine, <a style="color: #0076df; font-weight: bold">Beijing Wu</a>, Arwa Kathiria, Seung Woo Cho, Maxwell R. Mumbach, Ava C. Carter, Maya Kasowski, Lisa A. Orloff, Viviana I. Risca, Anshul Kundaje, Paul A. Khavari, Thomas J. Montine, William J. Greenleaf & Howard Y. Chang
                            </div>

                            <div class="periodical">
                                Nat Methods. 2017 Oct;14(10):959-962. doi: 10.1038/nmeth.4396.
                            </div>

                            <div class="links">
                                <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                                <a href="https://wubeijing.github.io/assets/pdf/nihms906979.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">Paper</a>
                            </div>

                            <!-- Hidden abstract block -->
                            <div class="abstract hidden">
                                <p align="left">We present Omni-ATAC, an improved ATAC-seq protocol for chromatin accessibility profiling that works across multiple applications with substantial improvement of signal-to-background ratio and information content. The Omni-ATAC protocol generates chromatin accessibility profiles from archival frozen tissue samples and 50-μm sections, revealing the activities of disease-associated DNA elements in distinct human brain structures. The Omni-ATAC protocol enables the interrogation of personal regulomes in tissue context and translational studies.</p>
                            </div>
                        </div>
                    </div>
                </li>
                
                <!-- publication 6 -->
                <li>
                    <div class="row">
                        <div class="col-sm-2 abbr">
                            <abbr class="badge" style="text-align: center; display: block">Nature Methods</abbr>
                        </div>

                        <div id="tenney2020lit" class="col-sm-8">
                            <a href="https://www.nature.com/articles/nmeth.4401" target="_blank">
                                <div class="title"><b>chromVAR: inferring transcription-factor-associated accessibility from single-cell epigenomic data</b></div>
                            </a>

                            <div class="author">
                                Alicia N. Schep, <a style="color: #0076df; font-weight: bold">Beijing Wu</a>, Jason D. Buenrostro & William J. Greenleaf
                            </div>

                            <div class="periodical">
                                Nat Methods. 2017 Oct;14(10):975-978. doi: 10.1038/nmeth.4401. 
                            </div>

                            <div class="links">
                                <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                                <a href="https://wubeijing.github.io/assets/pdf/nihms896446.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">Paper</a>
                            </div>

                            <!-- Hidden abstract block -->
                            <div class="abstract hidden">
                                <p align="left">Single-cell ATAC-seq (scATAC) yields sparse data that make conventional analysis challenging. We developed chromVAR <a href="http://www.github.com/GreenleafLab/chromVAR" target="_blank">(http://www.github.com/GreenleafLab/chromVAR)</a>, an R package for analyzing sparse chromatin-accessibility data by estimating gain or loss of accessibility within peaks sharing the same motif or annotation while controlling for technical biases. chromVAR enables accurate clustering of scATAC-seq profiles and characterization of known and <i>de novo</i> sequence motifs associated with variation in chromatin accessibility.</p>
                            </div>
                        </div>
                    </div>
                </li>

                <!-- publication 7 -->
                <li>
                    <div class="row">
                        <div class="col-sm-2 abbr">
                            <abbr class="badge" style="text-align: center; display: block">Genome Biology</abbr>
                        </div>

                        <div id="tenney2020lit" class="col-sm-8">
                            <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-1133-7" target="_blank">
                                <div class="title"><b>Single-cell epigenomic variability reveals functional cancer heterogeneity</b></div>
                            </a>

                            <div class="author">
                                Ulrike M. Litzenburger, Jason D. Buenrostro, <a style="color: #0076df; font-weight: bold">Beijing Wu</a>, Ying Shen, Nathan C. Sheffield, Arwa Kathiria, William J. Greenleaf & Howard Y. Chang
                            </div>

                            <div class="periodical">
                                Genome Biol. 2017 Jan 24;18(1):15. doi: 10.1186/s13059-016-1133-7.
                            </div>

                            <div class="links">
                                <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                                <a href="https://wubeijing.github.io/assets/pdf/13059_2016_Article_1133.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">Paper</a>
                            </div>

                            <!-- Hidden abstract block -->
                            <div class="abstract hidden">
                                <p align="left"><b>Background:</b> Cell-to-cell heterogeneity is a major driver of cancer evolution, progression, and emergence of drug resistance. Epigenomic variation at the single-cell level can rapidly create cancer heterogeneity but is difficult to detect and assess functionally.</p>
                                <p align="left"><b>Results:</b> We develop a strategy to bridge the gap between measurement and function in single-cell epigenomics. Using single-cell chromatin accessibility and RNA-seq data in K562 leukemic cells, we identify the cell surface marker CD24 as co-varying with chromatin accessibility changes linked to GATA transcription factors in single cells. Fluorescence-activated cell sorting of CD24 high versus low cells prospectively isolated GATA1 and GATA2 high versus low cells. GATA high versus low cells express differential gene regulatory networks, differential sensitivity to the drug imatinib mesylate, and differential self-renewal capacity. Lineage tracing experiments show that GATA/CD24hi cells have the capability to rapidly reconstitute the heterogeneity within the entire starting population, suggesting that GATA expression levels drive a phenotypically relevant source of epigenomic plasticity.</p>
                                <p align="left"><b>Conclusion:</b> Single-cell chromatin accessibility can guide prospective characterization of cancer heterogeneity. Epigenomic subpopulations in cancer impact drug sensitivity and the clonal dynamics of cancer evolution.</p>
                                <p align="left"><b>Keywords:</b> Cancer stem cells; Gene expression noise; Open chromatin.</p>
                            </div>
                        </div>
                    </div>
                </li>

                <h2 class="year">2016</h2>
                <!-- publication 8 -->
                <li>
                    <div class="row">
                        <div class="col-sm-2 abbr">
                            <abbr class="badge" style="text-align: center; display: block">Nature Genetics</abbr>
                        </div>

                        <div id="tenney2020lit" class="col-sm-8">
                            <a href="https://www.nature.com/articles/ng.3646" target="_blank">
                                <div class="title"><b>Lineage-specific and single-cell chromatin accessibility charts human hematopoiesis and leukemia evolution</b></div>
                            </a>

                            <div class="author">
                                M. Ryan Corces, Jason D. Buenrostro, <a style="color: #0076df; font-weight: bold">Beijing Wu</a>, Peyton G. Greenside, Steven M. Chan, Julie L. Koenig, Michael P. Snyder, Jonathan K. Pritchard, Anshul Kundaje, William J. Greenleaf, Ravindra Majeti & Howard Y. Chang
                            </div>

                            <div class="periodical">
                                Nat Genet. 2016 Oct;48(10):1193-203. doi: 10.1038/ng.3646.
                            </div>

                            <div class="links">
                                <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                                <a href="https://wubeijing.github.io/assets/pdf/nihms804249.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">Paper</a>
                            </div>

                            <!-- Hidden abstract block -->
                            <div class="abstract hidden">
                                <p align="left">We define the chromatin accessibility and transcriptional landscapes in 13 human primary blood cell types that span the hematopoietic hierarchy. Exploiting the finding that the enhancer landscape better reflects cell identity than mRNA levels, we enable 'enhancer cytometry' for enumeration of pure cell types from complex populations. We identify regulators governing hematopoietic differentiation and further show the lineage ontogeny of genetic elements linked to diverse human diseases. In acute myeloid leukemia (AML), chromatin accessibility uncovers unique regulatory evolution in cancer cells with a progressively increasing mutation burden. Single AML cells exhibit distinctive mixed regulome profiles corresponding to disparate developmental stages. A method to account for this regulatory heterogeneity identified cancer-specific deviations and implicated HOX factors as key regulators of preleukemic hematopoietic stem cell characteristics. Thus, regulome dynamics can provide diverse insights into hematopoietic development and disease.</p>
                            </div>
                        </div>
                    </div>
                </li>

                <h2 class="year">2015</h2>
                <!-- publication 9 -->
                <li>
                    <div class="row">
                        <div class="col-sm-2 abbr">
                            <abbr class="badge" style="text-align: center; display: block">Nature</abbr>
                        </div>

                        <div id="tenney2020lit" class="col-sm-8">
                            <a href="https://www.nature.com/articles/nature14590" target="_blank">
                                <div class="title"><b>Single-cell chromatin accessibility reveals principles of regulatory variation</b></div>
                            </a>

                            <div class="author">
                                Jason D. Buenrostro, <a style="color: #0076df; font-weight: bold">Beijing Wu</a>, Ulrike M. Litzenburger, Dave Ruff, Michael L. Gonzales, Michael P. Snyder, Howard Y. Chang & William J. Greenleaf
                            </div>

                            <div class="periodical">
                                Nature. 2015 Jul 23;523(7561):486-90. doi: 10.1038/nature14590.
                            </div>

                            <div class="links">
                                <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                                <a href="https://wubeijing.github.io/assets/pdf/nihms694863.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">Paper</a>
                            </div>

                            <!-- Hidden abstract block -->
                            <div class="abstract hidden">
                                <p align="left">Cell-to-cell variation is a universal feature of life that affects a wide range of biological phenomena, from developmental plasticity to tumour heterogeneity. Although recent advances have improved our ability to document cellular phenotypic variation, the fundamental mechanisms that generate variability from identical DNA sequences remain elusive. Here we reveal the landscape and principles of mammalian DNA regulatory variation by developing a robust method for mapping the accessible genome of individual cells by assay for transposase-accessible chromatin using sequencing (ATAC-seq) integrated into a programmable microfluidics platform. Single-cell ATAC-seq (scATAC-seq) maps from hundreds of single cells in aggregate closely resemble accessibility profiles from tens of millions of cells and provide insights into cell-to-cell variation. Accessibility variance is systematically associated with specific trans-factors and cis-elements, and we discover combinations of trans-factors associated with either induction or suppression of cell-to-cell variability. We further identify sets of trans-factors associated with cell-type-specific accessibility variance across eight cell types. Targeted perturbations of cell cycle or transcription factor signalling evoke stimulus-specific changes in this observed variability. The pattern of accessibility variation in cis across the genome recapitulates chromosome compartments de novo, linking single-cell accessibility variation to three-dimensional genome organization. Single-cell analysis of DNA accessibility provides new insight into cellular variation of the 'regulome'.</p>
                            </div>
                        </div>
                    </div>
                </li>

                <!-- publication 10 -->
                <li>
                    <div class="row">
                        <div class="col-sm-2 abbr">
                            <abbr class="badge" style="text-align: center; display: block">Current Protocols</abbr>
                        </div>

                        <div id="tenney2020lit" class="col-sm-8">
                            <a href="https://currentprotocols.onlinelibrary.wiley.com/doi/10.1002/0471142727.mb2129s109" target="_blank">
                                <div class="title"><b>ATAC-seq: A Method for Assaying Chromatin Accessibility Genome-Wide</b></div>
                            </a>

                            <div class="author">
                                Jason D. Buenrostro, <a style="color: #0076df; font-weight: bold">Beijing Wu</a>, Howard Y. Chang & William J. Greenleaf
                            </div>

                            <div class="periodical">
                                Curr Protoc Mol Biol. 2015 Jan 5;109:21.29.1-21.29.9. doi: 10.1002/0471142727.mb2129s109.
                            </div>

                            <div class="links">
                                <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                                <a href="https://wubeijing.github.io/assets/pdf/nihms653929.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">Paper</a>
                            </div>

                            <!-- Hidden abstract block -->
                            <div class="abstract hidden">
                                <p align="left">This unit describes Assay for Transposase-Accessible Chromatin with high-throughput sequencing (ATAC-seq), a method for mapping chromatin accessibility genome-wide. This method probes DNA accessibility with hyperactive Tn5 transposase, which inserts sequencing adapters into accessible regions of chromatin. Sequencing reads can then be used to infer regions of increased accessibility, as well as to map regions of transcription-factor binding and nucleosome position. The method is a fast and sensitive alternative to DNase-seq for assaying chromatin accessibility genome-wide, or to MNase-seq for assaying nucleosome positions in accessible regions of the genome.</p>
                            </div>
                        </div>
                    </div>
                </li>

                <h2 class="year">2014</h2>
                <!-- publication 11 -->
                <li>
                    <div class="row">
                        <div class="col-sm-2 abbr">
                            <abbr class="badge" style="text-align: center; display: block">Nature</abbr>
                        </div>

                        <div id="tenney2020lit" class="col-sm-8">
                            <a href="https://pubmed.ncbi.nlm.nih.gov/25164749/" target="_blank">
                                <div class="title"><b>Regulatory analysis of the C. elegans genome with spatiotemporal resolution</b></div>
                            </a>

                            <div class="author">
                                Carlos L. Araya, Trupti Kawli, Anshul Kundaje, Lixia Jiang, <a style="color: #0076df; font-weight: bold">Beijing Wu</a>, Dionne Vafeados, Robert Terrell, Peter Weissdepp, Louis Gevirtzman, Daniel Mace, Wei Niu, Alan P. Boyle, Dan Xie, Lijia Ma, John I. Murray, Valerie Reinke, Robert H. Waterston & Michael Snyder
                            </div>

                            <div class="periodical">
                                Nature. 2014 Aug 28;512(7515):400-5. doi: 10.1038/nature13497.
                            </div>

                            <div class="links">
                                <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                                <a href="https://wubeijing.github.io/assets/pdf/nihms691114.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">Paper</a>
                            </div>

                            <!-- Hidden abstract block -->
                            <div class="abstract hidden">
                                <p align="left">Discovering the structure and dynamics of transcriptional regulatory events in the genome with cellular and temporal resolution is crucial to understanding the regulatory underpinnings of development and disease. We determined the genomic distribution of binding sites for 92 transcription factors and regulatory proteins across multiple stages of Caenorhabditis elegans development by performing 241 ChIP-seq (chromatin immunoprecipitation followed by sequencing) experiments. Integration of regulatory binding and cellular-resolution expression data produced a spatiotemporally resolved metazoan transcription factor binding map. Using this map, we explore developmental regulatory circuits that encode combinatorial logic at the levels of co-binding and co-expression of transcription factors, characterizing the genomic coverage and clustering of regulatory binding, the binding preferences of, and biological processes regulated by, transcription factors, the global transcription factor co-associations and genomic subdomains that suggest shared patterns of regulation, and identifying key transcription factors and transcription factor co-associations for fate specification of individual lineages and cell types.</p>
                            </div>
                        </div>
                    </div>
                </li>
                
                <h2 class="year">2013</h2>
                <!-- publication 12 -->
                <li>
                    <div class="row">
                        <div class="col-sm-2 abbr">
                            <abbr class="badge" style="text-align: center; display: block">PLoS Genetics</abbr>
                        </div>

                        <div id="tenney2020lit" class="col-sm-8">
                            <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1003325" target="_blank">
                                <div class="title"><b>Roles of the developmental regulator unc-62/Homothorax in limiting longevity in <i>Caenorhabditis elegans</i></b></div>
                            </a>

                            <div class="author">
                                Eric L. Van Nostrand, Adolfo Sánchez-Blanco, <a style="color: #0076df; font-weight: bold">Beijing Wu</a>, Andy Nguyen, Stuart K. Kim
                            </div>

                            <div class="periodical">
                                PLoS Genet. 2013;9(2):e1003325. doi: 10.1371/journal.pgen.1003325.
                            </div>

                            <div class="links">
                                <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                                <a href="https://wubeijing.github.io/assets/pdf/pgen.1003325.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">Paper</a>
                            </div>

                            <!-- Hidden abstract block -->
                            <div class="abstract hidden">
                                <p align="left">The normal aging process is associated with stereotyped changes in gene expression, but the regulators responsible for these age-dependent changes are poorly understood. Using a novel genomics approach, we identified HOX co-factor <i>unc-62</i> (Homothorax) as a developmental regulator that binds proximal to age-regulated genes and modulates lifespan. Although <i>unc-62</i> is expressed in diverse tissues, its functions in the intestine play a particularly important role in modulating lifespan, as intestine-specific knockdown of <i>unc-62</i> by RNAi increases lifespan. An alternatively-spliced, tissue-specific isoform of <i>unc-62</i> is expressed exclusively in the intestine and declines with age. Through analysis of the downstream consequences of <i>unc-62</i> knockdown, we identify multiple effects linked to aging. First, <i>unc-62</i> RNAi decreases the expression of yolk proteins (vitellogenins) that aggregate in the body cavity in old age. Second, <i>unc-62</i> RNAi results in a broad increase in expression of intestinal genes that typically decrease expression with age, suggesting that <i>unc-62</i> activity balances intestinal resource allocation between yolk protein expression and fertility on the one hand and somatic functions on the other. Finally, in old age, the intestine shows increased expression of several aberrant genes; these UNC-62 targets are expressed predominantly in neuronal cells in developing animals, but surprisingly show increased expression in the intestine of old animals. Intestinal expression of some of these genes during aging is detrimental for longevity; notably, increased expression of insulin <i>ins-7</i> limits lifespan by repressing activity of insulin pathway response factor DAF-16/FOXO in aged animals. These results illustrate how <i>unc-62</i> regulation of intestinal gene expression is responsible for limiting lifespan during the normal aging process.</p>
                            </div>
                        </div>
                    </div>
                </li>

                <h2 class="year">2011</h2>
                <!-- publication 13 -->
                <li>
                    <div class="row">
                        <div class="col-sm-2 abbr">
                            <abbr class="badge" style="text-align: center; display: block">Biophysical Journal</abbr>
                        </div>

                        <div id="tenney2020lit" class="col-sm-8">
                            <a href="https://www.cell.com/biophysj/fulltext/S0006-3495(11)00339-0" target="_blank">
                                <div class="title"><b>Tryptophan-lipid interactions in membrane protein folding probed by ultraviolet resonance Raman and fluorescence spectroscopy</b></div>
                            </a>

                            <div class="author">
                                Katheryn M. Sanchez, Guipeun Kang, <a style="color: #0076df; font-weight: bold">Beijing Wu</a>, Judy E. Kim
                            </div>

                            <div class="periodical">
                                Biophys J. 2011 May 4;100(9):2121-30. doi: 10.1016/j.bpj.2011.03.018.
                            </div>

                            <div class="links">
                                <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                                <a href="https://wubeijing.github.io/assets/pdf/PIIS0006349511003390.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">Paper</a>
                            </div>

                            <!-- Hidden abstract block -->
                            <div class="abstract hidden">
                                <p align="left">Aromatic amino acids of membrane proteins are enriched at the lipid-water interface. The role of tryptophan on the folding and stability of an integral membrane protein is investigated with ultraviolet resonance Raman and fluorescence spectroscopy. We investigate a model system, the β-barrel outer membrane protein A (OmpA), and focus on interfacial tryptophan residues oriented toward the lipid bilayer (trp-7, trp-170, or trp-15) or the interior of the β-barrel pore (trp-102). OmpA mutants with a single tryptophan residue at a nonnative position 170 (Trp-170) or a native position 7 (Trp-7) exhibit the greatest stability, with Gibbs free energies of unfolding in the absence of denaturant of 9.4 and 6.7 kcal/mol, respectively. These mutants are more stable than the tryptophan-free OmpA mutant, which exhibits a free energy of unfolding of 2.6 kcal/mol. Ultraviolet resonance Raman spectra of Trp-170 and Trp-7 reveal evolution of a hydrogen bond in a nonpolar environment during the folding reaction, evidenced by systematic shifts in hydrophobicity and hydrogen bond markers. These observations suggest that the hydrogen bond acceptor is the lipid acyl carbonyl group, and this interaction contributes significantly to membrane protein stabilization. Other spectral changes are observed for a tryptophan residue at position 15, and these modifications are attributed to development of a tryptophan-lipid cation-π interaction that is more stabilizing than an intraprotein hydrogen bond by ∼2 kcal/mol. As expected, there is no evidence for lipid-protein interactions for the tryptophan residue oriented toward the interior of the β-barrel pore. These results highlight the significance of lipid-protein interactions, and indicate that the bilayer provides more than a hydrophobic environment for membrane protein folding. Instead, a paradigm of lipid-assisted membrane protein folding and stabilization must be adopted.</p>
                            </div>
                        </div>
                    </div>
                </li>

                <h2 class="year">2010</h2>
                <!-- publication 14 -->
                <li>
                    <div class="row">
                        <div class="col-sm-2 abbr">
                            <abbr class="badge" style="text-align: center; display: block">Science</abbr>
                        </div>

                        <div id="tenney2020lit" class="col-sm-8">
                            <a href="https://www.science.org/doi/10.1126/science.1196914?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed" target="_blank">
                                <div class="title"><b>Integrative Analysis of the <i>Caenorhabditis elegans</i> Genome by the modENCODE Project</b></div>
                            </a>

                            <div class="author">
                                Mark B. Gerstein, Zhi John Lu, Eric L. Van Nostrand, Chao Cheng, Bradley I. Arshinoff, Tao Liu, Kevin Y. Yip, Rebecca Robilotto, Andreas Rechtsteiner, Kohta Ikegami, Pedro Alves, Aurelien Chateigner, Marc Perry, Mitzi Morris, Raymond K. Auerbach, Xin Feng, Jing Leng, Anne Vielle, Wei Niu, Kahn Rhrissorrakrai, Ashish Agarwal, Roger P. Alexander, Galt Barber, Cathleen M. Brdlik, Jennifer Brennan, Jeremy Jean Brouillet, Adrian Carr, Ming-Sin Cheung, Hiram Clawson, Sergio Contrino, Luke O. Dannenberg, Abby F. Dernburg, Arshad Desai, Lindsay Dick, Andréa C. Dosé, Jiang Du, Thea Egelhofer, Sevinc Ercan, Ghia Euskirchen, Brent Ewing, Elise A. Feingold, Reto Gassmann, Peter J. Good, Phil Green, Francois Gullier, Michelle Gutwein, Mark S. Guyer, Lukas Habegger, Ting Han, Jorja G. Henikoff, Stefan R. Henz, Angie Hinrichs, Heather Holster, Tony Hyman, A. Leo Iniguez, Judith Janette, Morten Jensen, Masaomi Kato, W. James Kent, Ellen Kephart, Vishal Khivansara, Ekta Khurana, John K. Kim, Paulina Kolasinska-Zwierz, Eric C. Lai, Isabel Latorre, Amber Leahey, Suzanna Lewis, Paul Lloyd, Lucas Lochovsky, Rebecca F. Lowdon, Yaniv Lubling, Rachel Lyne, Michael MacCoss, Sebastian D. Mackowiak, Marco Mangone, Sheldon McKay, Desirea Mecenas, Gennifer Merrihew, David M Miller 3rd, Andrew Muroyama, John I. Murray, Siew-Loon Ooi, Hoang Pham, Taryn Phippen, Elicia A Preston, Nikolaus Rajewsky, Gunnar Rätsch, Heidi Rosenbaum, Joel Rozowsky, Kim Rutherford, Peter Ruzanov, Mihail Sarov, Rajkumar Sasidharan, Andrea Sboner, Paul Scheid, Eran Segal, Hyunjin Shin, Chong Shou, Frank J. Slack, Cindie Slightam, Richard Smith, William C. Spencer, E. O. Stinson, Scott Taing, Teruaki Takasaki, Dionne Vafeados, Ksenia Voronina, Guilin Wang, Nicole L. Washington, Christina M. Whittle, <a style="color: #0076df; font-weight: bold">Beijing Wu</a>, Koon-Kiu Yan, Georg Zeller, Zheng Zha, Mei Zhong, Xingliang Zhou, modENCODE Consortium; Julie Ahringer, Susan Strome, Kristin C. Gunsalus, Gos Micklem, X. Shirley Liu, Valerie Reinke, Stuart K. Kim, LaDeana W. Hillier, Steven Henikoff, Fabio Piano, Michael Snyder, Lincoln Stein, Jason D. Lieb, Robert H. Waterston
                            </div>

                            <div class="periodical">
                                Science. 2010 Dec 24;330(6012):1775-87. doi: 10.1126/science.1196914. 
                            </div>

                            <div class="links">
                                <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
                                <a href="https://wubeijing.github.io/assets/pdf/science.1196914.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">Paper</a>
                            </div>

                            <!-- Hidden abstract block -->
                            <div class="abstract hidden">
                                <p align="left">We systematically generated large-scale data sets to improve genome annotation for the nematode Caenorhabditis elegans, a key model organism. These data sets include transcriptome profiling across a developmental time course, genome-wide identification of transcription factor-binding sites, and maps of chromatin organization. From this, we created more complete and accurate gene models, including alternative splice forms and candidate noncoding RNAs. We constructed hierarchical networks of transcription factor-binding and microRNA interactions and discovered chromosomal locations bound by an unusually large number of transcription factors. Different patterns of chromatin composition and histone modification were revealed between chromosome arms and centers, with similarly prominent differences between autosomes and the X chromosome. Integrating data types, we built statistical models relating chromatin, transcription factor binding, and gene expression. Overall, our analyses ascribed putative functions to most of the conserved genome.</p>
                            </div>
                        </div>
                    </div>
                </li>


                </ol>
            </div>
        </div>
            
    </div>

    <!-- Footer -->
    <footer class="fixed-bottom">
        <div class="container mt-0">
            &copy; Powered by <a href="http://jekyllrb.com/" target="_blank">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio">al-folio</a> theme. Additional schema modifications contributed by <a href="https://iftenney.github.io/">others</a>.
            Hosted by <a href="https://pages.github.com/" target="_blank">GitHub Pages</a>.
        </div>
    </footer>
</body>

<!-- jQuery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg==" crossorigin="anonymous"></script>

<!-- Bootsrap & MDB scripts -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/2.4.4/umd/popper.min.js" integrity="sha512-eUQ9hGdLjBjY3F41CScH3UX+4JDSI9zXeroz7hJ+RteoCaY+GP/LDoM8AO+Pt+DRFw3nXqsjh9Zsts8hnYv8/A==" crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha512-M5KW3ztuIICmVIhjSqXe01oV2bpe248gOxqmlcYrEzAvws7Pw3z6BK0iGbrwvdrUQUhi3eXgtxp5I8PDo9YfjQ==" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/js/mdb.min.js" integrity="sha512-Mug9KHKmroQFMLm93zGrjhibM2z2Obg9l6qFG2qKjXEXkMp/VDkI4uju9m4QKPjWSwQ6O2qzZEnJDEeCw0Blcw==" crossorigin="anonymous"></script>

<!-- Load Common JS -->
<script src="/assets/js/common.js"></script>

<!-- Load DarkMode JS -->
<script src="/assets/js/dark_mode.js"></script>

</html>